app/assets/stylesheets/base/fonts.scss

```diff
@@ -141,7 +141,7 @@ body a {
 }
 
 .buttonsubmit {
```

Comment:
What do you think about putting a name like this: `button-submit`, in my opinion, this way is better to increase the legibility, I saw that you put all the class names like this, can you change the others too?